.github/workflows/build.yml

```diff
@@ -56,6 +56,31 @@ jobs:
           annotate_only: true
           include_passed: true
           report_paths: '**/TEST-*.xml'
+  open-api-static-analysis:
+    name: OpenApi Static Analysis Tests
+    runs-on: ubuntu-latest
+    env:
+      OPEN_API_TEST: "true"
+    steps:
+      - name: checkout code
+        uses: actions/checkout@v5.0.0
+      - name: setup java
+        uses: actions/setup-java@v5.0.0
+        with:
+          distribution: 'temurin'
+          java-version: 11
+          cache: 'gradle'
+      - name: build and test
+        id: thebuild
+        run: ./gradlew :cwms-data-api:test --tests "cwms.cda.api.OpenApiDocTest" --info --init-script init.gradle
+        continue-on-error: true
```

Comment:
remove this and set the below to "always"

We've tweaked the repo to have specific "required builds" this one for now is informational, but we still want it to show up as "failed" to draw eyes.